desktop/renderer-app/src/pages/PeriodicRoomDetailPage/index.tsx

```diff
@@ -42,7 +42,7 @@ export const PeriodicRoomDetailPage = observer<{}>(function PeriodicRoomDetailPa
     }, [periodicUUID, roomStore]);
 
     if (!periodicInfo || !rooms) {
-        return <LoadingPage />;
+        return <LoadingPage timeMS={3 * 1000} />;
```

Comment:
please leave some comment. It will take some time to understand

Reply:
yes